src/blockchain_state_channel_v1.proto

```diff
@@ -72,6 +72,38 @@ message blockchain_state_channel_purchase_v1 {
     bytes hotspot = 2;
     bytes packet_hash = 3;
     Region region = 4;
+    blockchain_state_channel_diff sc_diff = 5;
+}
+
+message blockchain_state_channel_diff {
+    bytes id = 1;
+    uint64 nonce = 2;
+    bytes signature = 3;
+    repeated blockchain_state_channel_diff_entry = 4;
+}
+
+enum blockchain_state_channel_diff_entry {
+    oneof entry {
+        blockchain_state_channel_diff_update_nonce nonce = 1;
```

Comment:
hoist this as a uint64 directly?

Reply:
Yeah that was what the nonce above was for